inner-tool/src/main/java/util/ObjectCodeGenerator.java

```diff
@@ -488,4 +489,14 @@ private static Class<?> narrow(Class<?> clazz, Object object) {
 
         return finder;
     }
+
+    private static String escape(String raw) {
+        return raw.replace("\\", "\\\\")
+                .replace("\"", "\\\"")
+                .replace("\b", "\\b")
+                .replace("\f", "\\f")
+                .replace("\n", "\\n\"\n  + \"")
```

Comment:
It's different from ValueJsonSerializer#escape, what for?

Reply:
it is for better readability - these new lines:

![image](https://github.com/chocovon/debug-variable-sl/assets/644816/d6d9dc82-0de3-4e8a-847f-35e94573bfcc)

Impossible with JSON